docs/application/dotnet/guides/wcircularui/overview.md

```diff
@@ -0,0 +1,94 @@
+# Wearable Circular UI
+
+The Wearable Circular UI project is a set of helpful extensions of the <a href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/" target="_blank">Xamarin.Forms</a> framework.
+The aim of this project is to develop an open source software motivate software developer to creating Tizen Wearable Xamarin Forms application more easily and efficiently.<br>
```

Comment:
The aim of this project is to develop an open source software and to motivate software developers to create Tizen Wearable Xamarin.Forms application more easily and efficiently.

Reply:
done